README.md

```diff
@@ -1,745 +1,148 @@
-about.
-
-While we have a variety of strategies for dealing with these issues, the most
-severe is simply to block offending applications. The main reason for you
-to be careful in using our request-naming tools is to prevent that from
-happening to your applications. We will do everything in our power to ensure
-that you have a good experience with New Relic even if your application is
-causing us trouble, but sometimes this will require manual intervention on the
-part of our team, and this can take a little while.
-
-## Contributions
-
-We owe a debt to all of the beta testers and users who have provided us with
-feedback, and in some cases significant pieces of code. (If you wish to
-contribute, please see [CONTRIBUTING.md](CONTRIBUTING.md) in this directory.) In particular, we're
-indebted to these people:
-
-* Hernan Silberman, for his work on the memcached instrumentation.
-* Jeff Howell &lt;jhowell@kabam.com&gt;, for coming up with a much simpler way
-  to instrument node-mongodb-native, as well as pointing out a problem with the
-  Connect instrumentation.
-
-## Recent changes
-
-Information about changes to the module are in [NEWS.md](NEWS.md).
-
-### Known issues:
-
-* New Relic for Node is only supported on Node.js 6 and newer. Some features
-  may behave differently between the supported versions of Node. The agent is
-  optimized for newer versions of Node.
-* There are irregularities around transaction trace capture and display. If you
-  notice missing or incorrect information from transaction traces, let us know.
-* There are <del>over 20,000</del> <del>30,000</del> <del>40,000</del> <ins>*A
-  LOT* of</ins> modules on npm. We can only instrument a tiny number of them.
-  Even for the modules we support, there are a very large number of ways to use
-  them. If you see data you don't expect on New Relic and have the time to
-  produce a reduced version of the code that is producing the strange data, it
-  will be used to improve the module and you will have the Node team's
-  gratitude.
-* The CPU and memory overhead incurred by New Relic for Node is relatively
-  minor (~1-10%, depending on how much of the instrumentation your apps end up
-  using). GC activity is significantly increased while the agent is active, due
-  to the large number of ephemeral objects created by metrics gathering.
-* When using Node's included clustering support, each worker process will open
-  its own connection to New Relic's servers, and will incur its own overhead
-  costs.
-
-### New Relic features available for other platforms not yet in Node.js
-
-* explain plans
-* thread profiling
-* X-ray transactions (depends on thread profiling)
-* capacity planning
+The NodeJS Agent also uses source code from third-party libraries. You can find full details on which libraries are used and the terms under which they are licensed in [the third-party notices document](/node-newrelic/blob/main/THIRD_PARTY_NOTICES.md).
```

Comment:
Third-party notices link appears broken in `rich diff` view.  This is the generated link:  `https://github.com/newrelic/node-newrelic/blob/main/node-newrelic/blob/main/THIRD_PARTY_NOTICES.md`
I think the path to the file should be relative to this one so this should work:  `THIRD_PARTY_NOTICES.md`